OTAnalytics/plugin_ui/customtkinter_gui/dummy_viewmodel.py

```diff
@@ -437,6 +448,7 @@ def load_tracks(self) -> None:
         print(f"Tracks files to load: {track_files}")
         track_paths = [Path(file) for file in track_files]
         self._application.add_tracks_of_files(track_files=track_paths)
+        self._application.intersect_tracks_with_sections()
```

Comment:
Is it necessary to run the intersection also in the next method (`load_sections`)?